backend/de.metas.printing/de.metas.printing.base/src/main/sql/postgresql/system/80-de.metas.printing/5620570_sys_gh12221_AddProductCategFilter.sql

```diff
@@ -0,0 +1,10 @@
+-- 2022-01-05T14:54:01.655813800Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Process_Para (AD_Client_ID,AD_Element_ID,AD_Org_ID,AD_Process_ID,AD_Process_Para_ID,AD_Reference_ID,ColumnName,Created,CreatedBy,Description,EntityType,FieldLength,Help,IsActive,IsAutocomplete,IsCentrallyMaintained,IsEncrypted,IsMandatory,IsRange,Name,SeqNo,Updated,UpdatedBy) VALUES (0,453,0,584953,542166,19,'M_Product_Category_ID',TO_TIMESTAMP('2022-01-05 16:54:01','YYYY-MM-DD HH24:MI:SS'),100,'Kategorie eines Produktes','U',0,'Identifiziert die Kategorie zu der ein Produkt gehört. Produktkategorien werden für Preisfindung und Auswahl verwendet.','Y','N','Y','N','Y','N','Produkt Kategorie',30,TO_TIMESTAMP('2022-01-05 16:54:01','YYYY-MM-DD HH24:MI:SS'),100)
```

Comment:
the description needs to explain wtf the parameter is about *with respect to this process*
Right now, when i hover over the parameter field "Product-Category", i am informed that it's "the category of a product" (BS). 
so which one shall i pick when i want to do Abrechnung? that info needs to be provided to me either in thge process help or in the field description.